src/rules/helpers.test.ts

```diff
@@ -0,0 +1,173 @@
+import {
+  ensureDirectoryStructureExists,
+  writeFile,
+} from '@metamask/utils/node';
+import path from 'path';
+
+import { directoryExists, fileExists } from './helpers';
+import { buildMetaMaskRepository, withinSandbox } from '../../tests/helpers';
+import { fail, pass } from '../execute-rules';
+
+describe('fileExists', () => {
+  it('passes if the given path refers to an existing file', async () => {
+    expect.assertions(1);
```

Comment:
What motivated this? We don't usually bother stating this explicitly.

Reply:
I made a change in `utils` which involved writing tests using `withinSandbox` and there was a suggestion there to use `expect.assertions`: https://github.com/MetaMask/utils/pull/148#discussion_r1362700719. I'm half and half on this, so if you don't think this is necessary, that's fine. It sounds like you disagree, so I've dropped this here but let me know if you want it back: [401adc3](https://github.com/MetaMask/module-lint/pull/39/commits/401adc3a411e2765111fbc4ba7dd07f0dd291b48)

Reply:
I just didn't understand the reason. I guess it's because the test might end up doing nothing if the helper function we're using here malfunctions? It wasn't totally clear.

Reply:
That was my interpretation, yes.